### PR TITLE
Updating project's configurations

### PR DESCRIPTION
**Pull request by lucis**
- Running `vtex setup` to [configure the project stuff](https://github.com/vtex/typescript/blob/master/docs/Getting%20Started.md).
- Removing `tslint` as it's deprecated.
- Replacing the `method.ts` middleware for the `method` helper function from `@vtex/api`.
- Removing unnecessary typing on the Service object.

(The workspace entry on `node/package.json` was not my call, and I think that there's no problem there)

### CODE REVIEW

File: node/package.json [hunk 1]
```diff
@@ -8,14 +8,13 @@
...
     "@types/node": "^12.0.0",
     "@types/ramda": "types/npm-ramda#dist",
-    "@vtex/api": "6.30.1",
+    "@vtex/api": "6.33.0",
     "@vtex/test-tools": "^1.0.0",
-    "tslint": "^5.14.0",
-    "tslint-config-vtex": "^2.1.0",
-    "typescript": "3.8.3"
+    "typescript": "3.8.3",
+    "vtex.service-example": "https://templo--extensions.myvtex.com/_v/private/typings/linked/v1/vtex.service-example@0.0.7+build1591194562/public/_types/react"
```
Comment (gris) [inline node/package.json:14]: Hum, this is a pain. It generates an error when first linking :(

File: node/index.ts [hunk 2]
```diff
@@ -34,20 +41,18 @@ declare global {
...
   // The shape of our State object found in `ctx.state`. This is used as state bag to communicate between middlewares.
-  interface State extends RecorderState{
+  interface State extends RecorderState {
     code: number
   }
 }
 
 // Export a service that defines route handlers and client options.
-export default new Service<Clients, State, ParamsContext>({
+export default new Service({
```
Comment (gris) [inline node/index.ts:50]: uououououou

File: node/package.json (hunk 1 above)
Comment (gris) [reply node/package.json:14]: I think it is better to remove it, don't you agree?

Comment (lucis) [reply node/package.json]: Done, @gris

**Pull request merged**